Fix banshee quest and wand of darkness attributes

Sparkz023 opened the pull request:
# Description

Fixed ID's for the 1st, 2nd and 6th seal. Added another key word to The Queen of the Banshee's NPC.

Small correction to items.xml to add crit damage and crit percent to wand of darkness. 

## Behaviour
### **Actual**

Currently, the first action will execute the revert function with a stone wall. The second seal (pearls) needs item ID's that aren't valid to access the teleporter. The sixth seal (fire camps) requires walls, instead of active fires to open the teleporter. The NPC has a keyword 'SEVENTH' that doesn't currently work.

The item 'Wand of Darkness' does not give critical chance or damage when equipped.

### **Expected**

The first action to remove the magic walls and gain entry to the quest area should be replaced with magic walls, not stone walls. The seals should require the correct item ID's to work. The NPC should respond to the keyword 'Seventh'. Wand of Darkness should give crit chance and damage.

## Fixes

*Changed ID's on the related scripts
*Added secondary keyword to The Queen of the Banshees
*Added crit chance and damage lines to wand of darkness in items.xml

## Type of change

Please delete options that are not relevant.

  - [X] Bug fix (non-breaking change which fixes an issue)
  - [ ] New feature (non-breaking change which adds functionality)
  - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
  - [ ] This change requires a documentation update

## How Has This Been Tested

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

  - [X] Test A - Replaced item ID's and completed the quest in full again on a local test server. Spoke to the NPC using both keywords (Seventh and Seventh Seal). Equipped and tested crit chance/damage on training machines.
  - [ ] Test B

**Test Configuration**:

  - Server Version: 1.3.1
  - Client: 12.86
  - Operating System: Ubuntu 20.2

## Checklist

  - [X] My code follows the style guidelines of this project
  - [X] I have performed a self-review of my own code
  - [X] I checked the PR checks reports
  - [X] I have commented my code, particularly in hard-to-understand areas
  - [X] I have made corresponding changes to the documentation
  - [X] My changes generate no new warnings
  - [X] I have added tests that prove my fix is effective or that my feature works

EnzzoCaaue commented:
nice work buddy!

Flyckks commented:
> -> Spawns NPCs through lua to improve performance
> -> Removed all npc spawns from spawns.xml file.
> 
> With the blessed help of @Flyckks

sniff sniff

Flyckks commented on data/globalevents/scripts/server/npcsSpawn.lua:
```diff
@@ -0,0 +1,17 @@
+
+function onStartup()
+    addEvent(_spawnNPCs, 1 * 1000)
+    print('Advanced NPC spawn system.')
```
`print('Advanced NPC spawn system | All NPCs will be loaded trough lib, instead of spawn.xml. Keep in mind, all your NPCs should be working, neighter the system will have a breakpoint..')`

lBaah replied:
Updated print message. Thank you.

MajestyOTBR commented:
Thanks for the contribution @lBaah and @Flyckks!

d0gw4r commented:
The npcs are walking trought chests and boxes, and also with not a range limit.
I realized that Quentin was walking on the streets. lol

Sparkz023 committed:
banshee quest

Fixed ID's for the 1st, 2nd and 6th seal. Added another key word to The Queen of the Banshee's NPC.

Small correction to items.xml

guispiller reviewed (approved):
Thanks for your contribution!

dudantas commented on data/npc/the_queen_of_the_banshees.lua:
```diff
@@ -59,7 +59,7 @@ local function creatureSayCallback(npc, creature, type, message)
 	end
 
-	if MsgContains(message, "seventh seal") then
+	if MsgContains(message, "seventh seal") or MsgContains(message, "seventh") then
```
With this pull request we do not need more of this change: https://github.com/opentibiabr/otservbr-global/pull/613

dudantas replied:
```suggestion
	if MsgContains(message, "seventh seal") then
```

majestyotbr committed:
Update the_queen_of_the_banshees.lua

Co-authored-by: Eduardo Dantas <eduardo.dantas@hotmail.com.br>
